plugins/woocommerce/changelog/fix-24244-custom-permalink

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: fix
+
+Redirect arbitrary term values wto the canonical value when product_cat is part of the product permalink structure
```

Comment:
I like the second one, but I think it should say "product" instead of "page":

```suggestion
Redirect to the canonical single product URL if the wrong category slug has been used.
```

Reply:
oh, yes!